mm2src/coins/utxo/utxo_common.rs

```diff
@@ -1432,11 +1429,7 @@ where
     };
     let outputs = vec![TransactionOutput { value, script_pubkey }];
     let fee = match req.fee {
-        Some(WithdrawFee::UtxoFixed { amount }) => {
-            let fixed = sat_from_big_decimal(&amount, decimals)?;
-            Some(ActualTxFee::Fixed(fixed))
-        },
-        Some(WithdrawFee::UtxoPerKbyte { amount }) => {
+        Some(WithdrawFee::UtxoPerKbyte { amount }) | Some(WithdrawFee::UtxoFixed { amount }) => {
```

Comment:
This change was made after a discussion with @artemii235 to avoid breaking change on withdraw